documentation/final-deliverables/pdf-report.md

```diff
@@ -2,110 +2,109 @@
+Mentees register through the Mentee Form, which can be accessed through an Organization's public website. Our system does not require mentees to have their own account. The Mentee Form prompts users to provide the skills that they want to be Mentored in as well as some basic information about themselves. Once the form has been submitted, a welcome email will be sent to the email address provided, informing the user that they will be notified once a Mentor has accepted their request. Based on the information provided on the form, MentorMatch will suggest Mentees to Mentors, considering common skills and time zones. After a Mentor accepts a Mentee request, that Mentee will receive an email containing information about the Mentor who accepted them, as well as the link which displays when the Mentor is available. From there, the two user types can correspond through email to arrange meetings.
 
-### DFD Level 1
+**An example of how Mentees interact with the system:**<br>
 
-* [DFD Level 1](https://github.com/jpabadir/pace-c/blob/requirements-milestone/documentation/dfd-level-1/dfd-full.jpg)
-* This DFD shows the system architecture within more detail.
-* It is encouraged to download the image to view it with more ease.
-* This DFD has the same three external entities as the previous one, but the mentor and mentee are duplicated in a few different places in order to avoid data flows crossing one another. Furthermore, it goes deeper into showing how accounts are actually created, and matches actually made. In addition, this DFD also shows the system's data stores, as opposed to its level 0 counterpart.
-* Any feature included in a red rectangle is to be completed for the Peer Review #2 milestone.
-* Any feature included in a dotted rectangle (or shown in a dotted fashion itself) is to be completed for the Final Project milestone.
+- After browsing through an Organization's website, I noticed that the Organization is registered on MentorMatch. Since I can relate to the Organization's mission, and I want to further my knowledge in the skillsets that they offer, I decide to register. The Mentee registration form is within the organization's website, and by following the tooltips and fixing any errors that are displayed, I submit the form, which notifies me that my submission has been successful, and that I'll be routed to MentorMatch in a new tab. Navigating to the new tab allows me to explore MentorMatch to see additional information. After checking my emails, I see one from MentorMatch welcoming me to the platform and letting me know that I'll be notified once a Mentor accepts my request.
+  By checking my emails, I'm eventually notified that a Mentor has accepted my request. Within this email, I can see the Mentor's name, their email address, a description about the individual, and a link to their calendar. Navigating to their calendar allows me to book an available time to meet.
```

Comment:
My bad, poor wording on my end. I meant that by going to the Google Calendar, an individual can click on an event on the calendar, and provide a message stating that they'd like to be mentored at that time (I've attached a screenshot of this below to help visualize what I'm referring to). I can definitely see how this wasn't reflected in how I originally phrased things! Mentors having to manually update their calendars to reflect booked time
slots is definitely a limitation, but [this](https://support.google.com/calendar/answer/190998?co=GENIE.Platform%3DDesktop&hl=en) provides a simple fix that we can note in our documentation later.

<img width="930" alt="Screen Shot 2021-03-23 at 3 39 11 PM" src="https://user-images.githubusercontent.com/20407302/112228080-ed51c080-8bed-11eb-9f06-141948c6f614.png">
<img width="659" alt="Screen Shot 2021-03-23 at 3 40 16 PM" src="https://user-images.githubusercontent.com/20407302/112228158-140ff700-8bee-11eb-8cca-4aab2b25b8fe.png">


For simplicity, would the following modification be alright?
```suggestion
  By checking my emails, I'm eventually notified that a Mentor has accepted my request. Within this email, I can see the Mentor's name, their email address, a description about the individual, and a link to their calendar. Navigating to their calendar allows me to view the Mentor's availability so that I can request a time to be mentored.
```

Reply:
Perfect!